### ISSUE TITLE
Events in month grid overlap, then don't when selecting new view and back again

### ISSUE THREAD

**Issue by gudlyf** [fullcalendar/fullcalendar#7060]
### Reduced Test Case

[REDUCED TEST CASE](https://gist.github.com/gudlyf/c27373b4784e26c87a5f9065be6fc0d8).

### Bug Description

When viewing the default day grid, events that happen on the same day overlap. When clicking another view (week, etc.), then back to the month view, the calendar looks as expected (events are not overlapping).

Expect to see the events not overlap at all, as in the 2nd screenshot.

### Screenshots

Overlap:
<img src="https://gist.github.com/gudlyf/c27373b4784e26c87a5f9065be6fc0d8/raw/b64c254838fd53d2c95aa515a0120efaf1d43279/calendar-overlap.jpg" />

No Overlap (desired result, which displays only after clicking another view, then back again):
<img src="https://gist.github.com/gudlyf/c27373b4784e26c87a5f9065be6fc0d8/raw/b64c254838fd53d2c95aa515a0120efaf1d43279/calendar-no-overlap.jpg" />

**Comment by arshaw** [comment]
Thanks for the report! Duplicate of:
#7089
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?
